bundle/orchestrate/dags/meltano.py

```diff
@@ -32,75 +32,152 @@
 }
 
 DEFAULT_TAGS = ["meltano"]
+PROJECT_ROOT = os.getenv("MELTANO_PROJECT_ROOT", os.getcwd())
+MELTANO_BIN = ".meltano/run/bin"
 
-project_root = os.getenv("MELTANO_PROJECT_ROOT", os.getcwd())
-
-meltano_bin = ".meltano/run/bin"
-
-if not Path(project_root).joinpath(meltano_bin).exists():
+if not Path(PROJECT_ROOT).joinpath(MELTANO_BIN).exists():
     logger.warning(
-        f"A symlink to the 'meltano' executable could not be found at '{meltano_bin}'. Falling back on expecting it to be in the PATH instead."
+        f"A symlink to the 'meltano' executable could not be found at '{MELTANO_BIN}'. Falling back on expecting it "
+        f"to be in the PATH instead. "
     )
-    meltano_bin = "meltano"
-
-
-result = subprocess.run(
-    [meltano_bin, "schedule", "list", "--format=json"],
-    cwd=project_root,
-    stdout=subprocess.PIPE,
-    universal_newlines=True,
-    check=True,
-)
-schedules = json.loads(result.stdout)
-
-for schedule in schedules:
-    logger.info(f"Considering schedule '{schedule['name']}': {schedule}")
+    MELTANO_BIN = "meltano"
+
+
+def _meltano_elt_generator(schedules):
+    """Generate singular dag's for each legacy Meltano elt task.
+
+    Args:
+        schedules (list): List of Meltano schedules.
+    """
+    for schedule in schedules:
+        logger.info(f"Considering schedule '{schedule['name']}': {schedule}")
+        if not schedule["cron_interval"]:
+            logger.info(
+                f"No DAG created for schedule '{schedule['name']}' because its interval is set to `@once`.",
+            )
+            continue
+
+        args = DEFAULT_ARGS.copy()
+        if schedule["start_date"]:
+            args["start_date"] = schedule["start_date"]
+
+        dag_id = f"meltano_{schedule['name']}"
```

Comment:
I didn't alter the id or tags for legacy ELT tasks because I wasn't sure if that might impact existing deployments.